blockchain/v0/reactor_test.go

```diff
@@ -92,7 +92,7 @@ func newBlockchainReactor(
 
 	// let's add some blocks in
 	for blockHeight := int64(1); blockHeight <= maxBlockHeight; blockHeight++ {
-		lastCommit := types.NewCommit(blockHeight-1, 0, types.BlockID{}, nil)
+		lastCommit := types.NewCommit(blockHeight-1, 0, types.BlockID{}, nil, []byte("bytes"))
```

Comment:
Can't we use sha256([]byte("bytes")) instead. It would be better if the constructor NewCommit wouldn't allow invalid inputs anyways.